pkg/antctl/command_definition.go

```diff
@@ -320,6 +327,80 @@ func respTransformer(obj interface{}) (interface{}, error) {
 	return target, nil
 }
 
+// getTableHeaders get the headers of the output table and set the configs for each column.
+func (cd *commandDefinition) getTableHeaders(obj common.TableOutput, tab *table.Writer) ([]string, error) {
+	headers, err := obj.GetTableHeader()
+	if err != nil {
+		return nil, fmt.Errorf("error when getting table header: %w", err)
+	}
+
+	tabConfigs := make([]table.ColumnConfig, 0)
+	tabHeaders := table.Row{}
+	// set the column configs
+	for _, arg := range headers {
+		colName := strings.ToUpper(arg)
+		tabHeaders = append(tabHeaders, colName)
+		tabConfig := table.ColumnConfig{
+			Name:     colName,
+			WidthMax: maxTableOutputColumnLength,
+		}
+		tabConfigs = append(tabConfigs, tabConfig)
+	}
+	(*tab).AppendHeader(tabHeaders)
+	(*tab).SetColumnConfigs(tabConfigs)
+
+	return headers, nil
+}
+
+// format the table output for "get" commands
+func (cd *commandDefinition) tableOutputForGetCommands(obj interface{}, writer io.Writer) error {
+	var list []common.TableOutput
+	if reflect.TypeOf(obj).Kind() == reflect.Slice {
+		s := reflect.ValueOf(obj)
+		if s.Len() == 0 {
+			var buffer bytes.Buffer
+			buffer.WriteString("\n")
+			if _, err := io.Copy(writer, &buffer); err != nil {
+				return fmt.Errorf("error when copy output into writer: %w", err)
+			}
+			return nil
+		}
+		for i := 0; i < s.Len(); i++ {
+			ele := s.Index(i)
+			list = append(list, ele.Interface().(common.TableOutput))
+		}
+	} else {
+		ele, _ := obj.(common.TableOutput)
+		list = []common.TableOutput{ele}
+	}
+
+	tab := table.NewWriter()
```

Comment:
I'd like to understand how much effort there is if not introducing a 3rd table format lib. I'm asking because I would still prefer the simple K8s style output (many antrea designs are in K8s style) and perhaps others have got used to that style too. 
Besides, it's easier to process the simple output with various text processing tools, e.g. wc, awk.